src/ray/object_manager/object_manager.h

```diff
@@ -416,6 +445,14 @@ class ObjectManager : public ObjectManagerInterface {
   /// remote object managers.
   std::unordered_map<ObjectID, PullRequest> pull_requests_;
 
+  /// A set of the chunks that are in the process of being pushed to remote
+  /// object managers. Each item consists of the ID of the remote object
+  /// manager, the ID of the object, and the index of the chunk.
+  std::vector<std::tuple<ClientID, ObjectID, uint64_t>> in_progress_pushes_;
+
+  /// The objects that are waiting to be pushed to other machines.
+  std::deque<std::pair<ObjectID, ClientID>> pending_pushes_;
+
```

Comment:
Add the size of this to debug string.

Reply:
Great suggestion, but fortunately I already did this :)